CMakeLists.txt

```diff
@@ -38,6 +38,7 @@ ecm_find_qmlmodule(QtPositioning 5.12)
 ecm_find_qmlmodule(org.kde.kcm 1.1)
 ecm_find_qmlmodule(org.kde.kirigami 2.10)
 
+find_package(libavif REQUIRED)
 find_package(libheif ${LIBHEIF_MIN_VERSION} REQUIRED)
```

Comment:
You'll need to either drop `libheif` as a dependency entirely or make it optional. If it remains mandatory, we can't ship it.

Reply:
Yeah, I will most likely drop the libheif code path.

Reply:
Sounds good to me. 👍🏾